### DISCUSSION TITLE
Texture projection use texture alpha doesn't work anymore

### DISCUSSION THREAD

**Discussion by marioepsley** [cables-gl/cables_docs#345] [! Welcome - readme first!]
Noticed that in normal cables (not dev.) the texture projection ' use texture alpha' doesn't seem to work anymore.

On dev it works but sadly on dev you can't stack the texture projections like you can in normal cables. have one after the other in a chain.

https://cables.gl/edit/ZhkO1d

**Comment by pandrr** [comment]
your patch is so slow i cant try it

**Reply by marioepsley** [reply]
Had a 100 in my cluster, resaved it with 10. should be quicker.

**Comment by pandrr** [comment]
on live it did not change, on dev the op was fixed.

what does it mean "you can't stack the texture prohections" ? the patch does not show that, i can add multiple textureProjections...

please be more clear!

**Comment by marioepsley** [comment]
It's fixed on dev now :) When I said 'stack' them, I meant chain them up. Before (on dev) when I had 2 texture projection ops one after another it seemed to crash. Not anymore.
